pom.xml

```diff
@@ -213,6 +213,11 @@
         <artifactId>spotbugs-annotations</artifactId>
         <version>4.8.4</version>
       </dependency>
+      <dependency>
+        <groupId>com.neovisionaries</groupId>
+        <artifactId>nv-i18n</artifactId>
```

Comment:
This is currently [unmaintained](https://github.com/TakahikoKawasaki/nv-i18n/issues/96). Since it's used only in few places, maybe later we can replace it with https://github.com/unicode-org/icu and https://github.com/unicode-org/icu/blob/main/icu4j/main/core/src/main/java/com/ibm/icu/util/ULocale.java ?